style/item.css

```diff
@@ -53,6 +71,32 @@
   cursor: pointer;
 }
 
+.forbidden-lands.item .tabs .tab-item {
+  padding: 5px;
+  text-align: center;
+}
+
+.forbidden-lands.item .tabs .tab-item.active {
+  color: #fff;
+  text-shadow: 0 0 10px #fff;
+}
+
+.forbidden-lands.item .sheet-body .tab {
+  margin-bottom: 8px;
+}
+
+.forbidden-lands.item .sheet-body .border {
+  padding: 8px;
+  margin-bottom: 8px;
+}
+
+.forbidden-lands.item .sheet-body .attributes,
+.forbidden-lands.item .sheet-body .supply {
```

Comment:
Preferential: Use the same layout for the supply and the crafting fields. Having labels in front in of the boxes and above in the other looks a bit off.